### ISSUE TITLE
[Error, GUI] Alive encountered an error 

### ISSUE THREAD

**Issue by hatelove** [CodeConnect/AliveFeedback#41]
error info:

```

---
[Error, GUI] Alive encountered an error 
[Error, GUI]  -> Object reference not set to an instance of an object. 
[Error, GUI] System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeConnect.ProjectOrange.AdornmentRegistry.CreateLaunchAdorner(MethodData methodInfo, IWpfTextView textView)
   at CodeConnect.ProjectOrange.Views.AliveAdornment.aliveIcon_Clicked(Object sender, MouseButtonEventArgs e) 

---
```

I just pass the first test case and create second test case. When I try to launch the second test case of my production code, it doesn't work.

PS: If I swtich to the other function to launch test case, it works again.

**Comment by AmadeusW** [comment]
We've revamped the GUI in version [0.6.2](https://github.com/CodeConnect/AliveFeedback/wiki/Changelog-0.6.2) and I'll check back soon to see if the problem still persists.

**Comment by AmadeusW** [comment]
This issue was resolved in 0.6.2
